### ISSUE TITLE
Still having pagination issues with custom post types

### ISSUE THREAD

**Issue by BenMcD0nald** [wp-graphql/wp-graphql#2133]
Hey,

I seem to be getting the same issue as demonstrated in #1714 but I keep reading that it should be fixed. 

As you can see from the screenshots below, when I go backwards the posts are not served in the same order as the original request. This makes navigation difficult  as subsequent requests are now offset and the results jump around between pages

I'm wonder if the fix isn't working with custom post types properly. I registered these for wpgraphql following this guide: https://www.wpgraphql.com/docs/custom-post-types/

Any thoughts?

Graphql version is Version 1.6.6
Wordpress: 5.8.1
PHP: 7.4.1 & 8 (tried both)

### Initial Load
![image](https://user-images.githubusercontent.com/30300743/139310478-fa8cbec3-d05a-43b0-ab4f-f639c54d6f27.png)

### Page Forward
![image](https://user-images.githubusercontent.com/30300743/139310606-c7f1190f-2ca2-4c55-b540-913f624f7916.png)

### Page Back
![image](https://user-images.githubusercontent.com/30300743/139310652-59c6301b-d938-4c47-a14c-ab881612ea93.png)

**Comment by mboynes** [comment]
@jasonbahl is there an open issue for the timestamp aspect of the issue in particular? I looked around but didn't see one. I'd be interested in helping to resolve it if you have more info.

**Comment by jasonbahl** [comment]
@mboynes I don't think there is 🤦🏻‍♂️ 

I'll create one now with some details.

**Comment by jasonbahl** [comment]
@mboynes I created this issue with a lot (maybe even too much) details: https://github.com/wp-graphql/wp-graphql/issues/2137

I also did some digging and might have a fix, but it's super late so I'm calling it a night. 

I'll test my fix more in the morning and post some more details and see if ya'll can verify that my changes work for your use cases. 

Talk soon! 🙏🏻

**Comment by mboynes** [comment]
Thanks @jasonbahl!

**Comment by jasonbahl** [comment]
@BenMcD0nald can you see if the changes applied in PR #2145 fix this for you?

**Comment by jasonbahl** [comment]
I've added some tests where posts are generated with identical titles and identical timestamps and with that set of data, I'm now able to paginate forward and backward with the results I would expect, where prior I was seeing the bug. 

I posted this video (https://www.youtube.com/watch?v=FnARcaOLKVI - also linked in that PR) walking through the bug and the behavior after my changes are applied.

**Comment by jasonbahl** [comment]
@BenMcD0nald @mboynes this fix is included in tomorrow's release v1.6.8

**Comment by BenMcD0nald** [comment]
Hey Jason,

Apologies for not getting back to you. I had my head stuck in something else development-wise and it took a while to circle back to being able to upload some dummy data. Funnily enough I did check into it earlier in the week and saw your test commit. I patched this in manually and it work and I was about to let you know when you sent the message about the release version.....thought I might as well wait and test that

So a long-winded way of saying yes I believe the patch works. Thanks so much for following up on it and resolving!!
